Tests/Test/ResponseTest.swift

```diff
@@ -101,11 +115,44 @@ class ResponseTest: XCTestCase {
     }
 
     func testBuildResponseWillDecodeOperators() {
-        let request = "GET /parameters?variable_1=Operators%20%3C%2C%20%3E%2C%20%3D%2C%20!%3D%3B%20%2B%2C%20-%2C%20*%2C%20%26%2C%20%40%2C%20%23%2C%20%24%2C%20%5B%2C%20%5D%3A%20%22is%20that%20all%22%3F"
+        let request = "GET /parameters?variable_1=Operators%20%3C%2C%20%3E%2C%20%3D%2C%20!%3D%3B%20%2B%2C%20-%2C%20*%2C%20%26%2C%20%40%2C%20%23%2C%20%24%2C%20%5B%2C%20%5D%3A%20%22is%20that%20all%22%3F\r\nConnection: Keep-Alive\r\n\r\n"
+
         let expectedResponse = "HTTP/1.1 200 OK\r\nContent-Length: 127\r\nContent-type: text/html\r\n\r\n<!DOCTYPE html><html><body><h1>variable_1 = Operators <, >, =, !=; +, -, *, &, @, #, $, [, ]: \"is that all\"?</h1></body></html>" 
         let parsedRequest = parser.parseRequest(request: request)
 
         XCTAssertEqual(expectedResponse, response.buildResponse(request: parsedRequest))
     }
 
+    func testBuildResponseWillReturnPOSTResponse() {
+        let request = "POST /form\r\nConnection: Keep-Alive\r\n\r\n\"My\"=\"Data\""
+        let expectedResponse = "HTTP/1.1 200 OK\r\nContent-Length: 0\r\nContent-type: text/html\r\n\r\n"
+
+        let parsedRequest = parser.parseRequest(request: request)
+
+        XCTAssertEqual(expectedResponse, response.buildResponse(request: parsedRequest))
+    }
+
+    func testBuildResponseWillReturnPUTResponse() {
+        let request = "PUT /form\r\nConnection: Keep-Alive\r\n\r\n\"My\"=\"Foo\""
+        let expectedResponse = "HTTP/1.1 200 OK\r\nContent-Length: 0\r\nContent-type: text/html\r\n\r\n"
+
+        let parsedRequest = parser.parseRequest(request: request)
+
+        XCTAssertEqual(expectedResponse, response.buildResponse(request: parsedRequest))
+    }
+
+    func testBuildResponseWillWritePOSTAndPUTToFile() {
+        let request1 = "POST /form\r\nConnection: Keep-Alive\r\n\r\n\"My\"=\"Data\""
+        let parsedRequest = parser.parseRequest(request: request1) 
+        response.buildResponse(request: parsedRequest)
+        let firstRequestData = readText() 
+
+        let request2 = "PUT /form\r\nConnection: Keep-Alive\r\n\r\n\"My\"=\"Foo\"" 
+        let parsedRequest2 = parser.parseRequest(request: request2) 
+        response.buildResponse(request: parsedRequest2)
+        let secondRequestData = readText()
+
+        XCTAssertNotEqual(firstRequestData, secondRequestData)
```

Comment:
The assertion doesn't match the test name. That can be addressed by better naming and splitting the test into at least one for POST and one for PUT.